Docs/source/usage/parameters.rst

```diff
@@ -2385,16 +2385,22 @@ Reduced Diagnostics
         * ``<reduced_diags_name>.reduction_type`` (`string`)
             The type of reduction to be performed. It must be either ``Maximum``, ``Minimum`` or
             ``Integral``.
-            ``Integral`` computes the spatial integral of the function defined in the parser by
-            summing its value on all grid points and multiplying the result by the volume of a
-            cell.
+            ``Integral`` computes the spatial surface or volume integral, depending on the choice
+            of the integration_type, of the function defined in the parser by summing its value on
+            all grid points and multiplying the result by the area or volume of a cell.
             Please be also aware that measuring maximum quantities might be very noisy in PIC
             simulations.
 
         The only output column is the reduced value.
 
         Note that the fields are averaged on the cell centers before the reduction is performed.
 
+        * ``<reduced_diags_name>.integration_type`` (`string`)
+           The type of integration to be performed. It must be either ``surface`` or ``volume``.
+
+        * ``<reduced_diags_name>.surface_normal`` (`string`)
+           The surface on which the surface integration is required. It must be either ``X``, ``Y`` or ``Z``.
```

Comment:
what about negatve direction surface normal?

Suppose we have a box, then we would have both positive and negative surface normals.

Reply:
Hi Reva, I haven't accounted for the direction here. 
I thought the field vector (e.g. E=i Ex+j Ey + k Ez) would account for the direction. 

If Ez is negative everywhere on the plane, then its value integrated over the plane will be negative. E.g. if on half of the surface of the Y (or XZ) plane, Ez is 1 and other half it is -1, surface integral will be zero. The normals are not really normals in the sense of directionality, but just a mean to indicate whether it is a X, Y, or a Z plane.

Reply:
Discussed offline about directionality for surface normal.

This is not for normal fields only, it sums the values of raw E-fields on a given plane.
If the user needs to compute surface integral normal to the plane, then they can do that in post-processing. So this will be clarified in the documentation

Reply:
Clarified this in the documentation. Please feel free to make changes.